src/zjs_i2c_ipm.c

```diff
@@ -0,0 +1,321 @@
+// Copyright (c) 2016, Intel Corporation.
+#ifndef QEMU_BUILD
+// Zephyr includes
+#include <i2c.h>
+#include <string.h>
+
+// ZJS includes
+#include "zjs_i2c.h"
+#include "zjs_ipm.h"
+#include "zjs_util.h"
+#include "zjs_buffer.h"
+
```

Comment:
How about add a block comment here about why we have separate implementation there - that it's specific to A101 w/ separate ARC core, etc.